### PR TITLE
[5096] Browse no longer supports upload

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/5096

### CODE REVIEW

File: ui/app/src/components/BrowseFilesDialog/BrowseFilesDialogContainer.tsx [hunk 1]
```diff
@@ -144,32 +157,74 @@ export function BrowseFilesDialogContainer(props: BrowseFilesDialogContainerProp
 
   const onCloseButtonClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => onClose(e, null);
 
-  return fetchingCurrentPathExists ? (
+  const onContextMenu = (element: Element) => {
+    console.log(element);
```
Comment (rart) [inline ui/app/src/components/BrowseFilesDialog/BrowseFilesDialogContainer.tsx]: x

File: ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx [hunk 2]
```diff
@@ -59,8 +60,17 @@ export function RenderTreeNode(props: RenderTreeNodeProps) {
...
+          ? (e) => {
+              e.preventDefault();
+              e.stopPropagation();
+              onContextMenu(e.target as Element);
+            }
+          : null
+      }
       label={
-        <div role="button" onClick={(e) => onLabelClick?.(e, node)}>
+        <div role="button" data-path={node.id} onClick={(e) => onLabelClick?.(e, node)}>
```
Comment (rart) [inline ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx]: What is the attribute for? Used where?

**Review by rart — CHANGES_REQUESTED** [review]
Let's remove the right click and context menu and leave the refresh and upload buttons at the top left of the items area. Search field on the right, smaller than it is now.

File: ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx (hunk 2 above)
Comment (aachavarria) [reply ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx]: this attribute is the path. it is used to open the upload dialog in the selected path

**Review by rart — APPROVED** [review]

**Pull request merged**